packages/antd/src/PollingUpdaterAnt.tsx

```diff
@@ -0,0 +1,157 @@
+    @observable protected _content: UIFragment | undefined;
+    @observable protected _failed = false;
+    @observable protected _errorMessage: string | undefined;
+    @observable protected _lastUpdate: number | undefined;
+    @observable protected _updateText: string | undefined;
+
+    protected _alive = false;
+    protected _polling: any;
+    protected _counter: any;
+
+    protected _update() {
+        this._pending = true;
+        this._lastUpdate = Date.now();
+        this.props.operation.update((errorMessage, content) => {
+            this._pending = false;
+            this._errorMessage = errorMessage;
+            this._failed = !!errorMessage;
+            this._content = content;
+        });
+    }
+
+    /**
+     * Start the polling and time counter
+     */
+    protected _awaken() {
+        if (this._alive) {
+            return;
+        }
+        const { updateFrequency } = this.props.operation;
+        this._alive = true;
+        this._update();
+        this._polling = setInterval(() => this._update(), updateFrequency * 1000);
+        this._counter = setInterval(() => {
+            const age = this._lastUpdate ? Math.round((Date.now() - this._lastUpdate) / 1000) : undefined;
+            this._updateText =
+                age !== undefined
+                    ? t(
+                          'pollingUpdater.updateAge.text',
+                          'Updated {{age}} seconds ago. Next update in {{left}} seconds.',
+                          { age, left: updateFrequency - age }
+                      )
+                    : t('pollingUpdater.neverLoaded', 'Never loaded.');
+        }, 1000);
+    }
+
+    /**
+     * Stop the polling and time counter
+     * @private
+     */
+    protected _ignore() {
+        if (!this._alive) {
+            return false;
+        }
+        this._alive = false;
+        clearInterval(this._polling);
+        clearInterval(this._counter);
+    }
+
+    constructor(props: PollingUpdaterProps) {
+        super(props);
+        this._update = this._update.bind(this);
+    }
+
+    render() {
+        const { operation } = this.props;
+        const { title, noPopup, noUpdateOnClick } = operation;
+
+        const coreContent = this._failed ? (
+            <Alert message={this._errorMessage} type={'warning'} />
+        ) : (
+            <span onClick={noUpdateOnClick ? undefined : this._update}>
+                {renderUIFragment(this._content)} {this._pending && <Spin />}
+            </span>
+        );
+        const clickText = noUpdateOnClick
+            ? undefined
+            : this._lastUpdate
+            ? t('pollingUpdater.clickToUpdate', 'Click to update now.')
+            : t('pollingUpdater.clickToLoad', 'Click to load.');
+        return noPopup ? (
+            coreContent
+        ) : (
+            <Popover
+                content={
+                    <span>
+                        {this._updateText} {clickText}
```

Comment:
The entire component will be re-rendered when the `_updateText` changes.

A better solution would be to create a separate popup component which only updates the UI when the popup is actually shown....